README.md

```diff
@@ -1,69 +1,32 @@
-```
-- Add hostname `https://localhost:3000` to your Google app's Authorized Javascript Origins
-
-## Start Development Server
-
-### App startup
-In a terminal window:
-```
-yarn install
-yarn dev
-```
-
-Most of the site should auto-compile and update when the source file is saved. One exception seems to be the socket server (code initialized by `/api/socket-keepalive`), which needs you to Ctrl-C the server and start `yarn dev` again.
-
-As updates are made, the site will re-compile on demand (page load). You may notice a delay when navigating between pages.
-
-### SSL Proxy
-There are parts of the application (specifically Google authentication, possibly future geolocation) that don't work on an insecure (http://) site. To get into a secure context, we'll use an SSL proxy. HTTPS traffic on port 9001 is forwarded to an HTTP server on port 3000.
-
-In another terminal window, start the proxy:
-```bash
-npx local-ssl-proxy --target 3000
-```
-
-### Connect to app
-Open a web browser to https://localhost:9001
-
-
-## Seed Data
+#### Add Seed Data to MongoDB
 The app is multi-tenant, and matches hostnames to organizations stored in the database. To be able to use the application, at least one organization must be created in the database. Below are two documents that should allow you to get up and running.
 
-The application does have integration with the organization's membership database (currently D4H). In order to set this up you will need an API key from your member database admin.
+The application also has integration with D4H, so you will need an API key, either generated yourself or from your member database admin.
+**Note that a D4H API key is equivalent to your password: it provides full access to personal information for every KCSARA member - even the bits that are marked ‘private’. Make sure you don't make this key public (e.g. by checking it in).**
+Try to get a key yourself:
+- Login to D4H
+- Click on the small head icon in the upper right and go to "My Settings"
+- Click on API access keys
+- Give your key a name and click "Generate API Access Key". You'll use this key in the next section. 
+
+If this fails, contact your database admin to ask for a key. 
 
-To fully test multi-tenancy you'll also need to be able to refer to the site by multiple host namess. It may be possible to do this with a combination of `localhost` and your computer's hostname. You may also want/need to add aliases to your local machine in your `/etc/hosts` file or router DNS configuration. These hostnames will be used in the seed data.
+To fully test multi-tenancy you'll also need to be able to refer to the site by multiple host names. It may be possible to do this with a combination of `localhost` and your computer's hostname. You may also want/need to add aliases to your local machine in your `/etc/hosts` file or router DNS configuration. These hostnames will be used in the seed data. However this is optional for an initial setup. 
```

Comment:
Would it make sense to switch
> `/etc/hosts` file

To just

> hosts file

I propose this because the Windows hosts file isn't located there, and the goal is that development should work on all platforms. It seems overkill to me to provide instructions for editing the hosts file on both *nix and Windows, as it's easy enough to look up.

(Though I'm not strongly against providing instructions for both platforms.)

Reply:
I left this because the path is nearly correct, even for Windows. Switched this to include the path for both Linux and Windows.

Reply:
Perfect. Thanks!